toolchain/sem_ir/node_kind.h

```diff
@@ -84,6 +84,11 @@ class NodeKind : public CARBON_ENUM_BASE(NodeKind) {
 // We expect the node kind to fit compactly into 8 bits.
 static_assert(sizeof(NodeKind) == 1, "Kind objects include padding!");
 
+// A definition of a node kind, including ancillary data such as the name to use
```

Comment:
How about we make this into a sentence?

```suggestion
// A definition of a node kind is a value of the NodeKind itself plus ancillary data such as the name to use
```

Reply:
Rephrased to include the "is a value of NodeKind" part.

(I found the suggested change as-is to make the comment harder for me to read -- generally I expect the first sentence in a class comment to be a noun phrase that could be preceded by "This class is...", in the same way that the first sentence in a class comment is missing its preceding "This function ...".)